systemtests/tests/python-dir-plugin-test/testrunner

```diff
@@ -43,23 +43,15 @@ status client
 status storage=File
 wait
 messages
-@#
-@# now do a restore
-@#
-@$out $tmp/log2.out
-wait
-restore client=bareos-fd fileset=PluginTest where=$tmp/bareos-restores select all done
-yes
-wait
-messages
 quit
 END_OF_DATA
 
 run_bareos
 check_for_zombie_jobs storage=File
 stop_bareos
 
-check_two_logs
-list="$(find ${BackupDirectory} -type f)"
-check_restore_only_files_diff $list
+if ! grep -Eq "^[bDirEventJobStart,bDirEventJobInit,bDirEventJobRun,bDirEventJobEnd]${1}" ${tmp}/test-plugin.log; then
+  estat=1
+fi
```

Comment:
Your grep statement looks wrong. How about the following one:

```suggestion
for i in bDirEventJobStart bDirEventJobInit bDirEventJobRun bDirEventJobEnd; do
  if ! grep "$i" ${tmp}/test-plugin.log; then
    set_error "Failed to find logged event $i"
  fi
done
```

Reply:
Beside the simple change to the "grep" line, I agree to this PR, but did ask @franku to also review this.
One other thing: please reformulate your 2. commit message summary to something like:

systemtests: added python-dir-plugin-test